### ISSUE TITLE
Chatterbox crashes in only if-statement nodes

### ISSUE THREAD

**Issue by Pumkin-dev** [JujuAdams/Chatterbox#34]
Hi Juju! I've wanted to make a node to decide wich node go into. Unfortunately, GM crashes when I do that
Yarn:
```
<<if !visited("Intro")>>
    [[Intro]]
<<else>>
    [[WakingUp]]
<<endif>>
```
Error at launch loading the file with the node:

```
___________________________________________
############################################################################################
ERROR in
action number 1
of Create Event
for object oChatterbox:

Push :: Execution Error - Variable Index [1] out of range [1] - -15.argument0(91,1)
 at gml_Script___chatterbox_compile_expression (line 846) -                 _token.a = _source_array[_t+1];
############################################################################################
gml_Script___chatterbox_compile_expression (line 846)
gml_Script___chatterbox_parse_expression (line 731) -     __chatterbox_compile_expression(_tokens);
gml_Script___chatterbox_compile (line 273) -                         _instruction.condition = __chatterbox_parse_expression(_remainder, false);
gml_Script___chatterbox_class_node (line 33) -     __chatterbox_compile(_substring_list, root_instruction);
gml_Script___chatterbox_class_source (line 48)
gml_Script_chatterbox_load_from_buffer (line 33)
gml_Script_chatterbox_load_from_file (line 47) -     return chatterbox_load_from_buffer(_filename, _buffer);
gml_Script_Setting_Chatterbox (line 5) -        chatterbox_load_from_file("intro.yarn")
gml_Object_oChatterbox_Create_0 (line 14) - Setting_Chatterbox();

```
Hope you'll find the cause!

**Comment by Pumkin-dev** [comment]
Oh I know what's the problem: Chatterbox doesn't like putting ! before visited. Without it, it works properly. Otherwise maybe you can do something about it!

**Comment by JujuAdams** [comment]
Fixed for v1.1.3 (https://github.com/JujuAdams/Chatterbox/commit/69b88209ad2a228daae1f6395a5dccba341f3fa0)
